src/VisualStudio/Core/Def/Workspace/VisualStudioActiveDocumentTracker.cs

```diff
@@ -317,7 +317,7 @@ private void TryInitializeTextBuffer()
                 _documentTracker.AssertIsForeground();
 
                 // NB: -5055 => __VSFPROPID12.VSFPROPID_IsDocDataInitialized available in 17.9
-                if (ErrorHandler.Succeeded(Frame.GetProperty((-5055), out var boxedIsDocDataInitialized)))
+                if (ErrorHandler.Succeeded(Frame.GetProperty(-5055, out var boxedIsDocDataInitialized)))
```

Comment:
is this intentionally a negative number?  that seems odd to me.  is this guaranteed this won't change?

Reply:
i would be fine waiting on this until we can reference the next public release of the API.